scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,853 @@
+#! /usr/bin/env python2
+"""
+usage: sonic_sku_create.py [-h] [-v] [-f FILE] [-m [MINIGRAPH_FILE]] [-b BASE]
+                           [-r] [-c [{new_sku_only,l2_mode_only}]] [-k HWSKU]
+                           [-p] [-vv]
+Create a new SKU
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -v, --version         show program's version number and exit
+  -f FILE, --file FILE  SKU definition from xml file. -f OR -m must be provided when creating a new SKU
+  -m [MINIGRAPH_FILE], --minigraph_file [MINIGRAPH_FILE]
+                        SKU definition from minigraph file. -f OR -m must be provided when creating a new SKU
+  -b BASE, --base BASE  SKU base definition
+  -r, --remove          Remove SKU folder
+  -c [{new_sku_only,l2_mode_only}], --cmd [{new_sku_only,l2_mode_only}]
+                        Choose action to preform (Generate a new SKU, Configure L2 mode, Both
+  -k HWSKU, --hwsku HWSKU
+                        SKU name to be used when creating a new SKU or for  L2 configuration mode
+  -p, --print           Print port_config.ini without creating a new SKU
+  -vv, --verbose        Verbose output
+
+
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import re
+import subprocess
+import swsssdk
+import traceback
+import sys
+import shutil
+import copy
+from collections import OrderedDict
+
+from tabulate import tabulate
+from lxml import etree as ET
+from lxml.etree import QName
+
+minigraph_ns = "Microsoft.Search.Autopilot.Evolution"
+minigraph_ns1 = "http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"
+INTERFACE_KEY = "Ethernet"
+
+test_envir = None
+try:
+	#test_envir = os.environ.get('UTILITIES_UNIT_TESTING')
+	test_envir = os.environ.get('USER')
```

Comment:
@madhanmellanox sonic-utilities is a separate repository - there is no special user-defined environment along with the special test mode. Please align script testing with some mocking concept, otherwise the Azure CI will fail this PR

Reply:
No, the same UTILITIES_UNIT_TESTING environment is used in dropconfig, dropstat, intfstat, intfutil, psushow, sfpshow etc.